content/python/concepts/built-in-functions/terms/isinstance/isinstance.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'isinstance()'
+Description: 'Returns True if the given object is the specified type. Otherwise the function will return False.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Booleans'
+  - 'Data Types'
+  - 'Development'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **isinstance()** function determines whether a given object is of a designated value type. If it is, the function will return `True`, otherwise the function will return `False`.
```

Comment:
```suggestion
The **`isinstance()`** function determines whether a given object is of a designated value type. If it is, the function will return `True`, otherwise the function will return `False`.
```